### DISCUSSION TITLE
How to modify/unprotected auto generated objects?

### DISCUSSION THREAD

**Discussion by IvanDrag0** [pgmodeler/pgmodeler#1907] [Q&A] [answered]
When creating relationships (and other objects) using the interface (right click -> New -> Relationship -> One-to-Many -> click on the source then click on the destination), everything works as expected, however, the auto variables that are created are protected and cannot be modified (for example, the ones in green in the following picture):

![image](https://github.com/user-attachments/assets/b8501cc2-19ae-4ab2-88c8-642869d3da6f)

As a result, they cannot be changed, renamed, etc. Is there a way to change and/or unprotect them so that I can modify them? I went over the guide, but I can't seem to find an option to do that.

**Comment by rkhaotix** [comment, marked as answer]
See here https://pgmodeler.io/support/docs/718-managing-relationship-generated-objects?v=1.1.0

**Reply by IvanDrag0** [reply]
Thank you!
